ui/README.md

```diff
@@ -0,0 +1,28 @@
+# Ui
```

Comment:
probably want to use the README from the dataflow master (just updated) as a template for the readme.
https://github.com/spring-cloud/spring-cloud-dataflow-ui/blob/master/README.md